### PR TITLE
DEV-239 update gdc_from_graph

### PR DESCRIPTION
**Pull request by qiaouchicago**
Use double for number type.

### CODE REVIEW

File: es-models/gdc_from_graph/settings.yaml [hunk 1]
```diff
@@ -26,6 +26,6 @@ analysis:
       filter:
       - lowercase
       type: custom
-index.mapping.nested_fields.limit: 100
-index.mapping.nested_objects.limit: 100000000
+index.mapping.total_fields.limit: 2000
 index.max_result_window: 100000000
+mapping.nested_fields.limit: 150
```
Comment (philtom-ctds) [inline es-models/gdc_from_graph/settings.yaml:31]: looks like you dropping an `index` prefix here.

File: es-models/gdc_from_graph/settings.yaml [hunk 2]
```diff
@@ -26,6 +26,6 @@ analysis:
       filter:
       - lowercase
       type: custom
-index.mapping.nested_fields.limit: 100
-index.mapping.nested_objects.limit: 100000000
+index.mapping.total_fields.limit: 2000
```
Comment (philtom-ctds) [inline es-models/gdc_from_graph/settings.yaml:29]: Can you explain why this setting is being increased?

File: es-models/gdc_from_graph/settings.yaml [hunk 3]
```diff
@@ -26,6 +26,6 @@ analysis:
       filter:
       - lowercase
       type: custom
-index.mapping.nested_fields.limit: 100
-index.mapping.nested_objects.limit: 100000000
```
Comment (philtom-ctds) [inline es-models/gdc_from_graph/settings.yaml:30]: Can you explain why this setting is being removed?

File: es-models/gdc_from_graph/case.mapping.yaml [hunk 4]
```diff
@@ -138,7 +138,7 @@ properties:
         normalizer: clinical_normalizer
```
Comment (philtom-ctds) [inline es-models/gdc_from_graph/case.mapping.yaml:138]: (This comment is not for this line. I just need a place to leave a comment.)

Some `number` fields in the dictionary that I don't see here:
```
clinical.yaml:  days_to_death:
clinical.yaml-    description: >-
clinical.yaml-      Time interval from a person's date of death to the date of initial pathologic
clinical.yaml-      diagnosis, represented as a calculated number of days. CDE: 3165475.
clinical.yaml-    type: number
```
```
clinical.yaml:  year_of_diagnosis:
clinical.yaml-    description: >-
clinical.yaml-      Numeric value to represent the year of an individual's initial pathologic diagnosis
clinical.yaml-      of cancer. CDE: 2896960.
clinical.yaml-    type: number
```
```
clinical.yaml:  age_at_diagnosis:
clinical.yaml-    description: >-
clinical.yaml-      Age in days at which a condition or disease was first diagnosed. CDE: 3225640.
clinical.yaml-      TCGA has the age in years with CDE 2006657, this is converted by ceil(age*365.25)
clinical.yaml-      to get days.
clinical.yaml-    type: number
```
```
pathology_detail.yaml-  greatest_tumor_dimension:
pathology_detail.yaml-    $ref:
pathology_detail.yaml-      - "_terms.yaml#/greatest_tumor_dimension/common"
pathology_detail.yaml:    type: number
```

Comment (anmaxvl) [reply es-models/gdc_from_graph/case.mapping.yaml:138]: we don't index `clinical` or `pathology_detail`. there's a ticket for adding indexing `pathology_detail`. but nothing for `clinical`. I'm not even sure we want to or have any `clinical` type data to index.

**Review by anmaxvl — APPROVED** [review]
didn't go through entire list, but changes in esbuild look correct and a spot check against dictionary also looked good to me

**Review by singergr — APPROVED** [review]
I skimmed the changes, and they seem reasonable. The passing tests in https://github.com/NCI-GDC/esbuild/pull/276 should confirm that this is good.

We will unfortunately need to go through and make all of the same updates to the case fields in the visualization (`_centric `) indices. 😢 We might want to file something for automatically regenerating those mappings based on the graph mappings.

**Comment by anmaxvl** [comment]
rebase?

**Comment by qiaouchicago** [comment]
> rebase?

nixie-knox?

**Comment by singergr** [comment]
> > rebase?
> 
> nixie-knox?
> develop?

develop. We are not planning to release this change in Nixie Knox.

**Pull request merged**